help/destinations/catalog/crm/salesforce.md

```diff
@@ -171,51 +186,55 @@ To correctly send your audience data from Adobe Experience Platform to the [!DNL
 
     * Add the following mapping between your XDM profile schema and your [!DNL Salesforce CRM] instance:
 
-    |XDM Profile Schema|[!DNL Salesforce CRM] Instance| Mandatory |
-    |---|---|---|
-    |`crmID`|`SalesforceId`|Yes|
+        |XDM Profile Schema|[!DNL Salesforce CRM] Instance| Mandatory |
+        |---|---|---|
+        |`crmID`|`SalesforceId`|Yes|
```

Comment:
I might have asked this before, but asking again. On the source field side, does the selected identity have to be crmID or can it be something else?

Reply:
Thanks for asking, have updated the section.